osidb/serializer.py

```diff
@@ -806,3 +806,47 @@ class Meta:
             "groups",
             "profile",
         ]
+
+
+class FlawCommentSerializer(
+    ACLMixinSerializer,
+    BugzillaSyncMixinSerializer,
+    TrackingMixinSerializer,
+    IncludeExcludeFieldsMixin,
+    IncludeMetaAttrMixin,
+):
+    """FlawComment serializer for use by flaw_comments endpoint"""
+
+    text = serializers.CharField(required=True)
+    type = serializers.CharField(max_length=100, required=False)
+    order = serializers.IntegerField(allow_null=True, required=True)
+
+    def create(self, validated_data):
+        """
+
+        Requires ATOMIC_REQUESTS so that temporary pending comment instances are
+        not left over after unsuccessful calls, as they could have unintended
+        consequences such as submitting long-forgotten comments to bugzilla later.
```

Comment:
I am not sure we have to mention this requirement all around. It is a general requirement for the whole OSIDB API to work properly that the requests are atomic.